src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -80,9 +80,8 @@ const AzureWarning = () => {
 
 
 const GooglePermissionsWarning = () => {
-  const warningMessage = 'Google is currently synchronizing its permissions with this workspace. This can take anywhere from a couple minutes ' +
-    'to a few hours in rare cases. Access to this workspace’s bucket and running analysis in workflows or notebooks may result in errors until ' +
-    'this synchronization is complete.'
+  const warningMessage = 'Google is currently synchronizing its permissions with this workspace. This can take several minutes ' +
```

Comment:
I would suggest "This can take several minutes, and in rare cases, several hours".

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/3856/commits/9e370bb1be7b2bf209ae530169e19bc6a9b353a8.